helm_deploy/values-preprod.yaml

```diff
@@ -6,11 +6,11 @@ generic-service:
 
   resources:
     requests:
-      cpu: 150m
```

Comment:
@steverendell do I need to reapply these resource limits?

Reply:
there were a couple OOMs in prod, so would be good though not vital